src/chrome/content/rules/Unblocked.xml

```diff
@@ -0,0 +1,31 @@
+<!--
+	The domain name for this site frequently changes
```

Comment:
hosts serve preloaded HSTS header, but is not yet in the list. Could you document that for future maintenance?